pkg/kn/commands/source/source_flags.go

```diff
@@ -0,0 +1,15 @@
+// Copyright © 2019 The Knative Authors
```

Comment:
can we move the flags to `pkg/kn/commands/flags/` ?

Reply:
I will suggest we keep source flags only in `kn/commands/source` folder, because it is `source` specific.

Reply:
I moved all sources flags to pkg/kn/commands/flags/